src/models/lineChart.js

```diff
@@ -251,7 +254,7 @@ nv.models.lineChart = function() {
                     .position({left: e.mouseX + margin.left, top: e.mouseY + margin.top})
                     .chartContainer(that.parentNode)
                     .valueFormatter(function(d,i) {
-                        return yAxis.tickFormat()(d);
+                        return d == null ? "N/A" : yAxis.tickFormat()(d);
```

Comment:
Wait, so here you return "N/A" but in the example html you override tickFormat to do the same, but that wouldn't ever happen since it doesn't call tickFormat if it's null... 

I wonder if we should do this instead so they can format nulls how they want?  Just a thought... 

    return d === null ? d : yAxis.tickFormat()(d);